### PR TITLE
Contact us

### PR DESCRIPTION
**Pull request by mombe090**
A PR https://wuri-guinea.atlassian.net/jira/software/projects/WUR/boards/1?selectedIssue=WUR-425

### CODE REVIEW

File: pre-registration-ui/src/app/core/contact-us/contact-us.component.html [hunk 1]
```diff
@@ -1,78 +1,130 @@
...
-                matInput 
-                placeholder="{{'contactUs.email' | translate}}" 
-                [formControlName]="formControlNames.email"/>
-                <mat-error *ngIf="!email.invalid"> {{'contactUs.mandatoryField' | translate}}</mat-error>
-        </mat-form-field>
-        <br />
+    <div class="wuri-card">
+        <div class="form">
+            <article>
+                <h4 class="title">Ecrivez-nous</h4>
```
Comment (wuritechteam) [inline pre-registration-ui/src/app/core/contact-us/contact-us.component.html:14]: Please translate

File: pre-registration-ui/src/app/core/contact-us/contact-us.component.html [hunk 2]
```diff
@@ -1,78 +1,130 @@
...
+                </form>
+            </article>
+        </div>
+        <div class="contact">
+            <article>
+                <div class="adresse">
+                    <p>
+                        <img src="assets/Envelope_ico.png" alt="adresse" width="40">
+                    </p>
+                    <h4 class="contact-title">Adresse</h4>
```
Comment (wuritechteam) [inline pre-registration-ui/src/app/core/contact-us/contact-us.component.html]: Please translate

File: pre-registration-ui/src/assets/i18n/fra.json [hunk 3]
```diff
@@ -253,20 +253,24 @@
...
 		"reason":"Raison",
 		"otherReason":"Autre Raison",
 		"message":"Message",
-		"maxChars":"(Max 250 Caractères)",
-		"mandatoryField":"Ce champs est requis",
+		"maxChars":"Max 250 Caractères MAXIMUM",
+		"mandatoryObjet":"Objet du message requis",
+		"mandatoryMsg":"Message requis",
+		"mandatoryName":"Prénom et nom requis",
+		"mandatoryEmail":"Adresse email requiseflwfel",
```
Comment (wuritechteam) [inline pre-registration-ui/src/assets/i18n/fra.json]: fix text

File: pre-registration-ui/src/app/core/contact-us/contact-us.component.html [hunk 4]
```diff
@@ -1,78 +1,130 @@
...
+                    </mat-form-field>
+                    <br />
 
+                    <button
+                            class="submit_button title"
+                            type="button"
+                            (click)="onSubmit()">
+                        {{'contactUs.send' | translate}}
+
+                        <!--<mat-spinner diameter="20" mode="indeterminate" style="display: inline"></mat-spinner>-->
```
Comment (elhadj21) [inline pre-registration-ui/src/app/core/contact-us/contact-us.component.html]: please  supprime le code mort

**Review by elhadj21 — APPROVED** [review]

**Pull request merged**